bindings/node/package.json

```diff
@@ -10,8 +10,8 @@
     "CHANGELOG.md",
     "lib",
     "src",
-    "build",
```

Comment:
@mbroadst @addaleax Hey, does this seem right to y'all? It's my understanding that the goal of prebuild is to either download or compile the package onsite, currently the build directory ships with a compiled version that matches the releaser's computer.

Reply:
LGTM,  this must have been left in from early testing. Do you need to include the `binding.gyp` for that matter?

Reply:
If a prebuild isn't available then it falls back on attempting to compile the module and this file is needed to do so.

Reply:
great point :facepalm: